src/App.jsx

```diff
@@ -4,15 +4,18 @@ import AddTodoForm from "./AddTodoForm.jsx";
 import React from "react";
 
 function App() {
-  const [newTodo, setNewTodo] = React.useState("");
   const [todoList, setTodoList] = React.useState([]);
 
+  function addTodo (newTodo) {
+    setTodoList([...todoList], newTodo)
+  }
```

Comment:
I'm not sure where the issue is, but new todo item is not added to the list. Will try to figure out

Reply:
solved